src/tv2-common/actions/executeAction.ts

```diff
@@ -1955,6 +1958,58 @@ async function executeActionClearGraphics<
 	})
 }
 
+async function executeActionClearTemaGraphics(context: ActionExecutionContext) {
+	await context.core.stopPiecesOnLayers([SharedSourceLayer.PgmGraphicsTema])
+	await context.core.insertPiece('current', {
+		enable: {
+			start: 'now',
+			duration: 3000
+		},
+		externalId: 'clearTemaGFX',
+		name: 'GFX Temaud',
+		sourceLayerId: SharedSourceLayer.PgmAdlibGraphicCmd,
+		outputLayerId: SharedOutputLayer.SEC,
+		lifespan: PieceLifespan.WithinPart,
+		content:
```

Comment:
This `content` is almost identical to the `content` of `executeActionClearAllGraphics`. It would be nice if they could reuse some of the same code.

You have several options to do so. One of them would be to extract it to a function that takes the thing being different as a parameter:
```
function createClearGraphicsContent(config: ..., vizElementTimelineObjectContent: ...): ... {
  return config.studio.GraphicsType === 'HTML'
    ?  {
        // HTML TimelineObjects here
    }
    :  {
      timelineObjects: [
        {
          // Viz Object here
          content: vizElementTimelineObjectContent
        }
      ]
    }
}
```
Above example is quite specific to the usecase at hand and not that flexible.
Another way to reduce duplication is just to extract the common parts which in this case is to extract how the `TimelineObjAbstractAny` is created, among other things.